src/biblenotify/ui/main.qml

```diff
@@ -283,10 +283,12 @@ Window {
     HomeView {
         id: homeView
         setTimeView: setTimeView
+        verseText: ""
+        verseReference: ""
```

Comment:
See the comment [here](https://github.com/BibleNotify/BibleNotifyDesktop/pull/16/files#diff-3e8b53b69c64b43736f570012160187574b8048d1cd9882a1f7d0b7fb298aeb5R12-R13).